.github/workflows/macos.yml

```diff
@@ -43,12 +43,6 @@ jobs:
             elixir-version: '1.14.3'
             os: 'macos-latest'
             version-type: 'strict'
-          - otp-version: '27'
-            os: 'macos-15'
-            version-type: 'strict'
-          - otp-version: '26'
-            os: 'macos-15'
-            version-type: 'strict'
```

Comment:
This shoulda been failing but wasn't. Guess it wasn't as strict as we intended...

Reply:
Oh, it was getting calculated as `maint-<otp-version>`.